spring-batch-docs/asciidoc/domain.adoc

```diff
@@ -170,8 +170,8 @@ image::{batch-asciidoc}images/job-stereotypes-parameters.png[Job Parameters, sca
 In the preceding example, where there are two instances, one for January 1st, and another
 for January 2nd, there is really only one `Job`, but it has two `JobParameter` objects:
 one that was started with a job parameter of 01-01-2017 and another that was started with
-a parameter of 01-02-2017. Thus, the contract can be defined as: `JobInstance` = `Job` +
-identifying `JobParameters`. This allows a developer to effectively control how a
+a parameter of 01-02-2017. Thus, the contract can be defined as: `JobInstance` = `Job`
+ + identifying `JobParameters`. This allows a developer to effectively control how a
```

Comment:
Without this space, the + sign is not rendered which makes the equation not readable.